gulpfile.js

```diff
@@ -12,5 +12,6 @@ var elixir = require('laravel-elixir');
  */
 
 elixir(function(mix) {
-    mix.less('app.less');
+    mix.less('admin.less');
+    mix.sass('main.scss');
```

Comment:
I think we'll need to add `resources/assets/js/main.js` to compile in here too. And potentially a task to copy the image assets over in to the public directory? It'd be good to go back to the Laravel 4 version of this repo and check what it's doing, since I think there were a bunch of custom Gulp tasks set up there.